### PR TITLE
Feat/map implementation

### PR DESCRIPTION
**Pull request by PuicanCristian**
Added routing and map functionality that return the location of a pin and the location of the center of the map. It uses react leaflet.

### CODE REVIEW

File: accessible-places-frontend/index.html [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.9.3/dist/leaflet.css"
+      integrity="sha256-kLaT2GOSpHechhsozzB+flnD+zUyjE2LlfWPgU04xyI="
+      crossorigin=""
+    />
+    <script
+      src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js"
+      integrity="sha256-WBkoXOwTeyKclOHuWtc+i2uENFpDZ9YPdf5Hf+D7ewM="
+      crossorigin=""
+    ></script>
```
Comment (dbalasoiu) [inline accessible-places-frontend/index.html:16]: repace with />

File: accessible-places-frontend/index.html [hunk 2]
```diff
@@ -3,6 +3,17 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React + TS</title>
```
Comment (dbalasoiu) [inline accessible-places-frontend/index.html]: add a title

File: accessible-places-frontend/index.html (hunk 1 above)
Comment (PuicanCristian) [reply accessible-places-frontend/index.html:16]: It returns an error is I try to put it <script/>

Comment (dbalasoiu) [reply accessible-places-frontend/index.html:16]: OK

**Pull request merged**